resources/lib/sosac.py

```diff
@@ -196,23 +206,22 @@ def list(self, url):
 
         return [self.dir_item(title="I failed", url="fail")]
 
+    def list_by_genres(self, url):
+        if "?" + GENRE_PARAM in url:
+            return self.list_xml_letter(url)
+        else:
+            result = []
+            page = util.request(url)
+            data = util.substr(page, '<select name=\"zanr\">', '</select')
```

Comment:
Lzoubek mě požádal o pull změn co jsem provedl, tak jsem je poslal. Kód je funkční a pokud vadí regexp, tak pak určitě nebudu přepisovat i další kód co ho používá a je v poslední oficiální verzi.
Nejsem programátor, snažil jsem se jen vyjít vstříc ostatním. Takže snad se najde někdo jiný, kdo tomu rozumí víc a přepíše to, já bohužel nejsem pythonista.